embedx/config.schema.json

```diff
@@ -2327,7 +2327,29 @@
       },
       "additionalProperties": false
     },
-    "ciphers": {
+    "passwordhash_comparators": {
```

Comment:
Why is this a separate top level config when we already have the `hashers` config? Can you add it there instead?

Reply:
moved to the `hashers` config